frontend/src/metabase/search/components/SearchResult.unit.spec.js

```diff
@@ -23,6 +29,61 @@ function collection({
   return collection;
 }
 
+describe("SearchResult > Tables", () => {
+  const setup = ({ tableOpts }) => {
```

Comment:
Ahh, sorry I just saw this. Didn't want to hold on to this PR for too long. I'm gonna approve it now, but could you move `setup` outside of `describe`? We always put it outer-most not inside any block. [Example](https://github.com/metabase/metabase/blob/4e734b0d4eaf8f6cb29bfb1e10a1d6c5c92c4cbe/frontend/src/metabase/components/AppBanner/AppBanner.unit.spec.tsx#L27-L53)

Reply:
The problem with that is there are other `describe` blocks in this file, that require different setups. This setup function is specific to setting up table search results and so it should be in the `describe` block that deals with these tests. What would you do if there needed to be another `describe` block with a different `setup` function, for let's say, model search results? You'd need two `setup` functions

Reply:
Thanks for the explanation.